src/ui/public/filter_manager/__tests__/filter_manager.js

```diff
@@ -114,13 +114,13 @@ describe('Filter Manager', function () {
     checkAddFilters(0, null, 3);
     expect(appState.filters).to.have.length(2);
 
-    let scriptedField = {name: 'scriptedField', scripted: true, script: 1};
+    let scriptedField = {name: 'scriptedField', scripted: true, script: 1, lang: 'painless'};
```

Comment:
So, I don't think this is a very good test, at least for this particular issue. I investigated adding a test that would actually send the query to ES but abandoned the endeavor as too time consuming (for the moment) since this is supposed to get into 5.0.  The original test didn't catch this bug so I find it somewhat useless (or at least geared towards verifying something else).  What if the ES syntax changes in the future?  We'd end up in the same situation and end up with a similar fix - updating both test and function.  So adding an extra, similar test is just an extra spot we'd have to manually adjust.

I can still add the test if you feel strongly about it, because there isn't any harm in it.  :)